third_party/java/arcs/build_defs/build_defs.bzl

```diff
@@ -54,6 +58,8 @@ arcs_manifest_bundle = _arcs_manifest_bundle
 
 arcs_manifest_json = _arcs_manifest_json
 
+arcs_manifest_parse_test = _arcs_manifest_parse_test
```

Comment:
Something about calling this a `test` rubs me the wrong way. It's more of a compile-time check. When I see test, I think more along the lines of a unit or integration test. WDYT?

Reply:
The analogy I was thinking of was a compile_test, which is a test to see that something compiles. However, we don't actually compile anything here (in that, we don't produce any output), we just parse the manifest and that's it. Hence, parse_test.